README.md

```diff
@@ -645,7 +645,7 @@ To enable this provider, pass `-a VulnerableCode` on the command line.
 
 The _evaluator_ is used to perform custom license policy checks on scan results. The rules to check against are
 implemented as scripts (currently Kotlin scripts, with a dedicated DSL, but support for other scripting can be added as
-well. See [example.rules.kts](./examples/example.rules.kts) for an example file.
+well. See [example.rules.kts](./examples/evaluator-rules/src/main/resources/example.rules.kts) for an example file.
```

Comment:
While on it you could also fix that the brace in the line above is never closed, and more importantly, mention that code completion for this file is supported and also link to the "evaluator-rules" example project. But the latter is probably better done in `file-rules-kts.md` which you also edit below.